cli_proton_python/sender.py

```diff
@@ -223,7 +225,7 @@ def send_message(self):
         if self.opts.log_stats == 'endpoints':
             utils.dump_event(self.event)
 
-        if self.opts.link_at_most_once and self.msg_sent_cnt == self.msg_total_cnt:
+        if self.opts.link_at_most_once and self.msg_sent_cnt - self.msg_rel_cnt == self.msg_total_cnt:
```

Comment:
msg_rel_cnt? (could ytou name i bit better? what is "rel"? But up to @pematous

Reply:
I second Michal's note. We have full names for all others above (total, confirmed, etc.. ) so let's keep the standard. Please use msg_release_cnt.

Reply:
OK.